contracts/protocol/libraries/logic/BridgeLogic.sol

```diff
@@ -125,7 +125,8 @@ library BridgeLogic {
     uint256 added = backingAmount + fee;
 
     reserveCache.nextLiquidityIndex = reserve.cumulateToLiquidityIndex(
-      IERC20(reserveCache.aTokenAddress).totalSupply(),
+      IERC20(reserveCache.aTokenAddress).totalSupply() +
```

Comment:
We can either use pass here the total amount, or compute the total amount within the `cumulateToLiquidityIndex` function. I think this approach is the most correct one and I'd add a short note in the function docs to clarify that the total available liquidity is not only the aToken totalSupply.

Reply:
For me, it is more correct as I put it, yes. Because the `cumulateToLiquidityIndex` this way is just a black box to accrue an amount (`feeToLP`) over a total.
About docs, I think this kind of stuff doesn't really belong to the code, should be in white paper/specs